models/base/scratch/snowplow_unified_base_events_this_run.sql

```diff
@@ -152,3 +157,24 @@ select
   {{ snowplow_unified.get_cwv_fields() }}
 {% endif -%}
 from base_query
+)
+
+SELECT *
+{% if target.type in ['bigquery','postgres', 'redshift'] and var('snowplow__enable_browser_context', false) and var('snowplow__enable_browser_context_2', false) %}
+
+  , coalesce(browser1__viewport, browser2__viewport) AS browser__viewport
```

Comment:
Here I see that browser__viewport is the name if they have both, in the past it was browser__viewport for v1, we should keep the same naming for backwards compatibility. Otherwise every time we have a major version change the original variables and namings will change.

Reply:
It is fine after all, I see the if else statements in the base events calls above.